ggshield/cmd/auth/auth_config.py

```diff
@@ -0,0 +1,195 @@
+from typing import Optional, Union
+
+import click
+
+from ggshield.core.config import (
+    AuthConfig,
+    Config,
+    InstanceConfig,
+    UnknownInstanceError,
+)
+
+
+DATETIME_FORMAT = "%Y-%m-%dT%H:%M:%SZ"
+
+FIELD_OPTIONS = ("default_token_lifetime",)
+
+
+@click.command()
+@click.pass_context
+def config_list_cmd(ctx: click.Context) -> int:
+    """
+    List all auth configurations. Example of output
+
+    [default]
+    default_token_lifetime: 14d
+    workspace: 23:
+    url: *dashboard.gitguardian.com*
+    token: *62890f237c703c92fbda8236ec2a055ac21332a46115005c976d68b900535fb5*
+    token_name: *my_token*
+    token_type: *personal_access_token*
+    expiry: *2022-02-23T12:34:56Z*
+    """
+    config: Config = ctx.obj["config"]
+
+    message = ""
+    for instance in config.auth_config.instances:
+
+        """
+        [default]
+        default_token_lifetime: 14d
+        workspace: 23:
+        url: *dashboard.gitguardian.com*
+        token: *62890f237c703c92fbda8236ec2a055ac21332a46115005c976d68b900535fb5*
+        token_name: *my_token*
+        token_type: *personal_access_token*
+        expiry: *2022-02-23T12:34:56Z*
+
+        """
+        instance_name = instance.name or instance.url
+
+        if instance.account is not None:
+            account_id = instance.account.account_id
+            token = instance.account.token
+            token_name = instance.account.token_name
+            expire_at = instance.account.expire_at
+            expiry = (
+                expire_at.strftime(DATETIME_FORMAT)
+                if expire_at is not None
+                else "never"
+            )
+        else:
+            account_id = token = token_name = expiry = "not set"  # type: ignore
+
+        message_lines = [
+            f"[{instance_name}]",
+            f"default_token_lifetime: {instance.default_token_lifetime}",
+            f"workspace: {account_id}:",
```

Comment:
I think this field should be called `account_id`.

Reply:
I am following the tech specs https://www.notion.so/gitguardian/ggshield-auth-9ffc8074e97e4c439805246867380074#c989f6c9170a4f1b8762fb734153fbb9

but I will check

Reply:
Indeed, I am curious if the different name is there on purpose.

Reply:
from Eric:
> account is not something understandable for the end user.
we must use workspace(_id?) unfortunately

Reply:
Uh, OK.